norent/forms.py

```diff
@@ -19,6 +20,11 @@ class Meta:
         model = JustfixUser
         fields = ('first_name', 'last_name')
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['first_name'].required = True
+        self.fields['last_name'].required = True
```

Comment:
Oops, this is unrelated to internationalization but I realized that we're not actually _requiring_ this information in our form!